systers_portal/static/css/style.css

```diff
@@ -247,6 +251,18 @@ table.decoration-none  tr a {
   margin: 30px 0px;
 }
 
+.text-indent35 {
+  text-indent: 35px;
+}
+
+/* Meetup Location Members Grid CSS
+-------------------------------------------------- */
+.crop_img {
+  width: 100px;
+  height: 100px;
+  overflow: hidden;
+}
+
```

Comment:
To deal with your squashing image issue, try setting max-width and max-height to 100px each and width and height both to auto and see how the image looks.